pkg/utils/status.go

```diff
@@ -6,35 +6,43 @@ import (
 )
 
 // InitialExperimentStatus fills up ExperimentStatus Structure with InitialValues
-func (expStatus *ExperimentStatus) InitialExperimentStatus(jobName string) {
-	expStatus.Name = jobName
+func (expStatus *ExperimentStatus) InitialExperimentStatus(expName, engineName string) {
+	expStatus.Name = expName
+	expStatus.Runner = engineName + "-runner"
+	expStatus.ExpPod = "Yet to be launched"
 	expStatus.Status = "Waiting for Job Creation"
-	expStatus.Verdict = "Waiting"
+	expStatus.Verdict = "N/A"
 	expStatus.LastUpdateTime = metav1.Now()
 }
 
 // AwaitedExperimentStatus fills up ExperimentStatus Structure with Running Status
-func (expStatus *ExperimentStatus) AwaitedExperimentStatus(experimentDetails *ExperimentDetails) {
-	expStatus.Name = experimentDetails.JobName
+func (expStatus *ExperimentStatus) AwaitedExperimentStatus(expName, engineName, experimentPodName string) {
+	expStatus.Name = expName
+	expStatus.Runner = engineName + "-runner"
+	expStatus.ExpPod = experimentPodName
 	expStatus.Status = "Running"
 	expStatus.Verdict = "Awaited"
 	expStatus.LastUpdateTime = metav1.Now()
 }
 
 // CompletedExperimentStatus fills up ExperimentStatus Structure with values chaosResult
-func (expStatus *ExperimentStatus) CompletedExperimentStatus(chaosResult *v1alpha1.ChaosResult, experimentDetails *ExperimentDetails) {
+func (expStatus *ExperimentStatus) CompletedExperimentStatus(chaosResult *v1alpha1.ChaosResult, engineName, experimentPodName string) {
 	//var currExpStatus v1alpha1.ExperimentStatuses
-	expStatus.Name = experimentDetails.JobName
+	expStatus.Name = chaosResult.Spec.ExperimentName
+	expStatus.Runner = engineName + "-runner"
+	expStatus.ExpPod = experimentPodName
 	expStatus.Status = "Execution Successful"
```

Comment:
experiment status in chaosengine `Completed` vs `Execution Successful`?

Reply:
Makes sense! Thanks